HyperV/scripts/gerrit-git-prep.sh

```diff
@@ -94,7 +90,7 @@ set -x
 
 if [[ ! -e .git ]]
 then
-    ls -a
+    ls -a && pwd
```

Comment:
no && plz. Do it on separate lines. And add an echo to know what is shown... As in what step and what is expected.